runtime/src/transaction_batch.rs

```diff
@@ -1,22 +1,37 @@
 use {
-    crate::bank::Bank,
-    solana_sdk::transaction::{Result, SanitizedTransaction},
-    std::borrow::Cow,
+    crate::bank::Bank, solana_sdk::transaction::Result,
+    solana_svm_transaction::svm_transaction::SVMTransaction,
 };
 
+pub enum OwnedOrBorrowed<'a, T> {
```

Comment:
Could not find something similar in std or core, but might have missed it.
If you're aware, please let me know 😄 

I don't want to add an additional generic parameter aside from the `Tx`, so trait like Borrow or Deref doesn't fit - although would work I think.

Reply:
my reflex comment is that the use of `Cow` is questionable in the first place, iirc. this is indicated by the fact ToOwned isn't used.

maybe try `Borrow<Target = impl SVMMessage>`?

Reply:
I think that might work, but would mean we need to have even another generic parameter.
Which would be fine if we were just creating the batches and using them - since rust is smart enough to tell what we're passing without specifying.
It just feels a bit burdensome when we pass the batches to functions, since now we need 2 generic parameters everywhere we pass `TransactionBatch` as an arg.

It seems fine to me to have a single type (non-lifetime) parameter, but having 2 is a lot - especially since it'd prevent us from using the convenient `TransactionBatch<impl SVMMessage>` notation.

Reply:
oh, thanks for further explnation.

in that case, we need to introduce another trait... dunno you like it or not: ddefffd134a7b6310cab5ad7cf207ce462cad5e9 (note that it merely compiles `solana-runtime` with `cfg[not(test)]`...)

as we transition the code base, i think we can get rid of the predicates (`<Tgt = SanitizedTransaction>`).

Reply:
Personally I prefer the `OwnedOrBorrowed` instead of some new trait